host/sgx/create.c

```diff
@@ -106,6 +106,21 @@ static bool _is_kss_supported()
         return true;
 }
 
+static bool _is_MISC_region_supported()
```

Comment:
```suggestion
static bool _is_misc_region_supported()
```
https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md#naming-conventions says:
> Use `lower_snake_case` for variable, member/field, and function names.